docs/textinput.md

```diff
@@ -128,11 +128,11 @@ Tells `TextInput` to automatically capitalize certain characters. This property
 
 ---
 
-### `autoCompleteType`
+### `autoComplete`
 
-Specifies autocomplete hints for the system, so it can provide autofill. On Android, the system will always attempt to offer autofill by using heuristics to identify the type of content. To disable autocomplete, set `autoCompleteType` to `off`.
+Specifies autocomplete hints for the system, so it can provide autofill. On Android, the system will always attempt to offer autofill by using heuristics to identify the type of content. To disable autocomplete, set `autoComplete` to `off`.
```

Comment:
I think we should add one of those platform tags since this is an android only property?

Reply:
Something like the green tag here: https://reactnative.dev/docs/next/text#datadetectortype-android

Reply:
@jeswinsimon In #2708 @luism3861 just added the missing platform labels for `TextImput` docs, it would be nice if you can rebase your PR and add the platform label as Luna suggested. 🙂